ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -453,7 +453,7 @@ private Void readVsc(String[] tokens) {
 
     public AmplNetworkReader readMetrics(Map<String, String> metrics) throws IOException {
         Objects.requireNonNull(metrics);
-        try (BufferedReader reader = new BufferedReader(new InputStreamReader(dataSource.newInputStream("_indic", "txt"), StandardCharsets.UTF_8))) {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(dataSource.newInputStream(dataSource.getMainFileName() + "_indic.txt"), StandardCharsets.UTF_8))) {
```

Comment:
In the other calls, you only append the suffix (e.g. not the extension): are you sure that it’s ok? Is there a unit test to ensure the code is correct?